public/js/eventsPage.js

```diff
@@ -8,8 +8,13 @@ const message = document.querySelector('#messages')
  *
  * @return event data
  */
-function getEvents() {
-    fetch('./api/getEvents', {
+function getEvents(search = false) {
+    let url = './api/getEvents'
+    if (search !== false) {
```

Comment:
Why did you use !== false rather than == true?.

Reply:
This is a strict comparison and makes more sense than using true (used when the input it blank)